test/fixtures/command_line/spec.json

```diff
@@ -10,28 +10,33 @@
         "VALIDATE_WORKSPACE": false
     },
     "extra_files": [
+        "examples/command_line/lib/BUILD",
         {
             "_": "macos-x86_64-min12.0-applebin_macos-darwin_x86_64-fastbuild-ST-f2aadcfcab57/bin/examples/command_line/lib/lib_impl.swift.modulemap",
             "t": "g"
         },
         "examples/command_line/lib/dir with space/lib.h",
+        "examples/command_line/tool/BUILD",
         "examples/command_line/tool/Info.plist",
         {
             "_": "macos-x86_64-min11.0-applebin_macos-darwin_x86_64-fastbuild-ST-8946c8252059/bin/examples/command_line/lib/lib_impl.swift.modulemap",
             "t": "g"
         },
+        "examples/command_line/Tests/BUILD",
         {
             "_": "macos-x86_64-min11.0-applebin_macos-darwin_x86_64-fastbuild-ST-8946c8252059/bin/examples/command_line/lib/lib_swift.swift.modulemap",
             "t": "g"
         },
+        "apple/testing/BUILD",
```

Comment:
Not sure what the best way is to avoid this from being included.

Reply:
Not a good way right now. We can open an issue to clean it up later, which I'll do when I refactor the input collection as part of fixing #228.

Reply:
Actually, we can just only include BUILD files for the project instead of the external repos:
```
if not target.label.workspace_root:
    # Add BUILD file here
```

Reply:
7aebba5